<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Vista_User</class>
 <widget class="QWidget" name="Vista_User">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Registrate</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>257</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Fecha de nacimiento</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>297</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Registro">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>377</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background-color:rgb(51, 255, 102);
color:black;
border:circular
}</string>
   </property>
   <property name="text">
    <string>Registrarse</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEmail">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Correo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Salir_2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>380</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="linePass">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Contraseña</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="linePass_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Repite Contraseña</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
